src/modules/hardwareWallet/hooks/useHWAccounts.js

```diff
@@ -27,7 +25,17 @@ const useHWAccounts = () => {
         setIsLoadingHWAccounts(true);
         try {
           const accounts = await getHWAccounts(currentHWDevice, getAccountName);
-          dispatch(setHWAccounts(accounts));
+          const uniqueAccounts = accounts.reduce((accum, account) => {
+            const indexOfAccount = accum.findIndex(
+              // eslint-disable-next-line max-nested-callbacks
+              (item) => item.metadata.address === account.metadata.address
+            );
+            if (indexOfAccount === -1) {
+              accum.push(account);
+            }
+            return accum;
+          }, []);
+          dispatch(setHWAccounts(uniqueAccounts));
```

Comment:
```suggestion
          const uniqueAccountsObject = accounts.reduce((result, account) => {
             result[account.metadata.address] = account
          }, {});
          
          dispatch(setHWAccounts(Object.values(uniqueAccountsObject)));
```

I feel this is more readable and easy to follow compared to the above

Reply:
https://github.com/LiskHQ/lisk-desktop/pull/4912/commits/c64b9fbd503f0b40d31cf12ea36667f8bda258a2